From d87b665c7e4c09bf713ef6f45ceb11a8aacbcf5e Mon Sep 17 00:00:00 2001
From: Aaron Shaw <shawaj@gmail.com>
Date: Tue, 10 Jan 2017 16:11:04 +0000
Subject: [PATCH] Add driver_name paramater

Add driver_name parameter for use with 5.1 passthrough audio in LibreElec and other Kodi OSs
---
 sound/soc/bcm/justboom-digi.c | 1 +
 1 file changed, 1 insertion(+)

--- a/sound/soc/bcm/justboom-digi.c
+++ b/sound/soc/bcm/justboom-digi.c
@@ -154,6 +154,7 @@ static struct snd_soc_dai_link snd_rpi_j
 /* audio machine driver */
 static struct snd_soc_card snd_rpi_justboom_digi = {
 	.name         = "snd_rpi_justboom_digi",
+	.driver_name  = "JustBoomDigi",
 	.owner        = THIS_MODULE,
 	.dai_link     = snd_rpi_justboom_digi_dai,
 	.num_links    = ARRAY_SIZE(snd_rpi_justboom_digi_dai),
